tutorial/part3/data_exploitability_pangeo.ipynb

```diff
@@ -8,6 +8,491 @@
    "metadata": {
```

Comment:
<p>Typo for my name: Michele Claus and GitHub username is @clausmichele</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/pangeo-openeo-BiDS-2023/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>